common/iso_config.go

```diff
@@ -140,9 +140,20 @@ func (c *ISOConfig) parseCheckSumFile(rd *bufio.Reader) error {
 	if err != nil {
 		return err
 	}
+
+	checksumurl, err := url.Parse(c.ISOChecksumURL)
+	if err != nil {
+		return err
+	}
+
+	relpath, err := filepath.Rel(filepath.Dir(checksumurl.Path), u.Path)
+	if err != nil {
+		return err
+	}
+
 	filename := filepath.Base(u.Path)
 
-	errNotFound := fmt.Errorf("No checksum for %q found at: %s", filename, c.ISOChecksumURL)
+	errNotFound := fmt.Errorf("No checksum for %q or %q found at: %s", filename, relpath, c.ISOChecksumURL)
```

Comment:
Not a huge fan of how this error message is formatted; shouldn't relpath be grouped with the checksumurl rather than with the filename?